src/lib/utils/testing/dom-tools.ts

```diff
@@ -73,6 +76,14 @@ function hasClass(element: any, className: string): boolean {
   return element.classList.contains(className);
 }
 
+function hasAttribute(element: any, attributeName: string): boolean {
+  return element.hasAttribute(attributeName);
```

Comment:
It doesn't look like most of these DOM utilities are doing much, aside from proxying the native functions with the exact same arguments.